### ISSUE TITLE
[Accessibility]: Notebooks read-only ARC 

### ISSUE THREAD

**Issue by marisak** [sourcegraph/sourcegraph#35670]
### Audit type

ARC Toolkit

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/35142

### Problem description

Per the ARC extension, the green checkmark icon in the Notebooks header doesn't have a label. From what I understand, the label is required because the icon is focusable. 

### Expected behavior

Either: 
- The icon should have a label 
- The icon should not be focusable 

### Additional details

![sourcegraph com_notebooks_Tm90ZWJvb2s6Mjk4 (1)](https://user-images.githubusercontent.com/16570574/169161570-667fd589-73c9-4eda-a33b-01256fd44cea.png)

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.
